nbcparser/main.go

```diff
@@ -1,40 +1,36 @@
 package main
 
 import (
-	"log"
+	"fmt"
 	"os"
 
 	"github.com/Azure/agentbaker/nbcparser/pkg/parser"
 )
 
-// input to this function will be the serialized JSON from userdata + custom data.
+// This script will take in the bootstrap contract in JSON format,
 // it will be deserialized to the contract that the VHD this binary will be on supports.
 // Parse will be called using that deserialized struct and output the generated cse_cmd to trigger the bootstrap process.
-// example usage:
-// to build: go build main.go.
-// to run: ./main testdata/test_nbc.json.
 //
 //nolint:gosec // generated cse_cmd.sh file needs execute permissions for bootstrapping
+const (
+	CSE_CMD = "cse_cmd.sh"
+)
+
 func main() {
 	if len(os.Args) < parser.MinArgs {
```

Comment:
nit(non-blocking): would prefer we use stdlib `flags` or `cobra` for flags rather than raw argv, can do that in another PR though

Reply:
modified to use `flag`